src/vtp/create_blank_ballot.py

```diff
@@ -27,16 +27,16 @@
 
 # Standard imports
 # pylint: disable=wrong-import-position
-import sys
 import argparse
 import logging
-from logging import info, debug
 import pprint
+import sys
+from logging import debug, info
```

Comment:
rats - I thought that was the way people wanted it to be.  Happy to make the change (emacs + tags-query-replace)

Reply:
pushed

Reply:
... and repaired.  FWIIW I think the reason the design idiom is to NOT import the print methods is that one also not use f statements within the parameter list.  There is an explicit pylint error on this.  The reason for the pylint error is that is best to use the old style %s style so that the args are not fully evaluated if not printed.  For example:

verify_ballot_receipt.py:70:12: W1203: Use lazy % formatting in logging functions (logging-fstring-interpolation)

So 99% of the logging statements where using f statements - and are not using lazing % syntax.

Reply:
> FWIIW I think the reason the design idiom is to NOT import the print methods is that one also not use f statements within the parameter list.

Hmm. I doubt that's the reason. The convention of not importing log methods goes back to before Python 3, so it is much older than f-strings (which were added in Python 3.6).

> The reason for the pylint error is that is best to use the old style %s style so that the args are not fully evaluated if not printed.

That's my understanding. See: https://docs.python.org/3/howto/logging-cookbook.html#formatting-styles for a clear explanation. (That doc explains that it _is_ possible with some support code to use brace-style formatting if you want to. But this still doesn't allow for f-string interpolation.)